packages/maskbook/src/database/Persona/Persona.db.ts

```diff
@@ -39,8 +46,10 @@ const db = createDBAccessWithAsyncUpgrade<PersonaDB, Knowledge>(
                 function v0_v1() {
                     db.createObjectStore('personas', { keyPath: 'identifier' })
                     db.createObjectStore('profiles', { keyPath: 'identifier' })
+                    db.createObjectStore('relations', { keyPath: ['linked', 'profile'] })
```

Comment:
No, to add a new object store, you must do a database upgrade otherwise old users will fail to upgrade.